app/Models/Device.php

```diff
@@ -324,13 +324,49 @@ public function getAttribs()
         return $this->attribs->pluck('attrib_value', 'attrib_type')->toArray();
     }
 
+    /**
+     * @param $domain  Domain which has to be parsed
+     * @param $record  DNS Record which should be searched
+     * @return array   List of matching records
+     */
+    public function getDnsRecord($record = 'A')
```

Comment:
this function doesnt belong in the Device model

Reply:
some ideas where to store function?

Reply:
Id suggest creating a new class under `LibreNMS/Util/`

Reply:
changed